ntbs-service-unit-tests/DataMigration/ImportValidatorTest.cs

```diff
@@ -127,26 +127,463 @@ public async Task ImportValidatorValidatesCollectionModels()
                 errorMessages);
         }
 
```

Comment:
There are definitely enough contact tracing tests here that it's worth considering moving them to their own test class

Reply:
Yep, pulled into a partial if that's ok, given they are all testing the same class?

Reply:
Yep, that's fine by me